bin/pocs_shell

```diff
@@ -74,6 +74,17 @@ class PocsShell(Cmd):
 
         return self.pocs.is_safe()
 
+    def do_display_config(self, *arg):
+        """ Display the config file for POCS """
+        try:
+            if self.pocs and self.pocs.config:
+                print_warning("No config file for POCS.")
+            else:
```

Comment:
I think the if and else cases have been swapped.

Reply:
Seems like I did it for both. Fixing now, thank you.